src/mtcp/restore_libc.c

```diff
@@ -268,10 +268,11 @@ TLSInfo_GetTidOffset(void)
 
     tid_offset = tmp - (char *)pthread_desc;
     if (tid_offset != STATIC_TLS_TID_OFFSET()) {
+      // NOTE: PRINTF can only handle strings upto 256 chars
       PRINTF("WARNING: tid_offset (%d) different from expected.\n"
              "  It is possible that DMTCP was compiled with a different\n"
              "  glibc version than the one it's dynamically linking to.\n"
-             "  Continuing anyway.  If this fails, please try again.",
+             "  If this fails, please try again.",
```

Comment:
But in this change, you lose the string "continuing anyway.  ".  A better solution is to split the `PRINTF()` into to separate `PRINTF()` commands.  Each of them can be passed a string of less than 256 characters.